presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -45,6 +45,12 @@ protected QueryRunner createQueryRunner()
         return createAccumuloQueryRunner(ImmutableMap.of());
     }
 
+    @Override
+    public void testCreateSchema()
+    {
+        // schema creation is not supported
```

Comment:
it's better `throw new SkipException`

Reply:
This is existing behavior, and we do this in many other places, so I'll leave it alone here. If we want to change the policy, we'll do it consistently instead of for one method.

Reply:
i believe both  are "existing behavior"

Reply:
eg https://github.com/prestosql/presto/commit/789027e972bd9b5013db18d9f2c2d659c110353d